proto/projects/project.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+package lavanet.lava.projects;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/lavanet/lava/x/projects/types";
+
+message Project {
+    string index = 1; // unique id that will be the combination of subscription address and project name, cannot be changed once created
+    string subscription = 2; // the subscription address that owns the project
+    string description = 3; // the description of the project for the users convinient
+    bool enabled = 4; // enabled flag
+    repeated ProjectKey project_keys = 5 [(gogoproto.nullable) = false];; // list of the projects keys
+    Policy policy = 6 [(gogoproto.nullable) = false];;
+    uint64 used_cu = 7;
+}
+
+message ProjectKey {
+    string  key = 1; // the address of the project key
+    
+    enum KEY_TYPE{
+        ADMIN = 0;
+        DEVELOPER = 1; 
+    }    
+
+    repeated KEY_TYPE types = 2 [(gogoproto.nullable) = false];; // the key type, determines the privilages of the key
+}
+
+message Policy {
+    repeated ChainPolicy chain_policies = 1 [(gogoproto.nullable) = false];;
+    uint64 geolocation_progile = 2;
+    uint64 total_cu_limit = 3;
+    uint64 epoch_cu_limit = 4;
+}
+
+message ChainPolicy {
```

Comment:
why not just "Chain". There is no policy related fields in this struct

Reply:
the policy is the chains in the ist and the apis that developers can use with this project